docker/ray-ml/Dockerfile

```diff
@@ -1,27 +1,29 @@
 ARG GPU
-FROM rayproject/ray:nightly"$GPU"
+ARG DOCKER_PREFIX
```

Comment:
Could `DOCKER_PREFIX` be defaulted to `rayproject`?

Reply:
Sure :)